waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -50,6 +55,12 @@ dbSchemaFileInProjectRootDir = dbRootDirInProjectRootDir </> dbSchemaFileInDbRoo
 dbMigrationsDirInDbRootDir :: Path' (Rel DbRootDir) (Dir DbMigrationsDir)
 dbMigrationsDirInDbRootDir = [reldir|migrations|]
 
+dbSchemaChecksumFileInDbMigrationsDir :: Path' (Rel DbMigrationsDir) File'
+dbSchemaChecksumFileInDbMigrationsDir = [relfile|prisma.schema-checksum|]
```

Comment:
I think internal files being noted as `.wasp` somehow makes sense! 👍🏻

Reply:
Yeah I like it!
Btw, isnt it `schema.prisma`, and not `prisma.schema`? I myself make this mistake many time and am never sure, but I just checked their docs and I see that they say `schema.prisma`.
I would suggest name `schema.prisma.wasp-checksum`. So it follows that convention of `somefile.txt.old` that I have seen sometimes on Linux. Or `foo.cpp.swp`. Usually it is the same name together with extension, and then suffix is added but starting with a dot.

Reply:
Oof woops on the swapping, thanks for catching. `schema.prisma.wasp-checksum` sounds good.

Reply:
Btw I wonder if we should still be putting it in the migrations dir? Now we can put it anywhere, right? Since we don't care about copying it up? Or actually, it would be better to have it outside of migrations dir, so it doesn't by accident get copied up? Hmmm not sure but asking questions.

Reply:
Yeah, I moved out of migrations dir when we wanted to keep purely local and now it just lives alongside `schema.prisma` as `.wasp/out/db/schema.prisma.wasp-checksum`